src/sshfs_mount/sshfs_mount.cpp

```diff
@@ -59,19 +62,50 @@ void check_sshfs_exists(mp::SSHSession& session)
     run_cmd(session, "which sshfs", error_handler);
 }
 
-void make_target_dir(mp::SSHSession& session, const std::string& target)
+// Split a path into existing and to-be-created parts.
+std::pair<std::string, std::string> get_path_split(mp::SSHSession& session, const std::string& target)
```

Comment:
Maybe we could make all these `QString`s so there's not so many `::fromStdString`, `.toStdString()`?

Reply:
I'd like to keep QString and QDir local to the split function. Note that if we convert some std::string to QString, we may need to use QString::toStdString in some places, thus we are in the same situation, I think.